vscode/src/utils.ts

```diff
@@ -258,13 +258,13 @@ export async function promptForWiFiInfo(): Promise<WiFiInfo> {
     "prompt": "Enter Wi-Fi SSID"
   };
   const ssid = await Window.showInputBox(ssidPromptOptions);
-  if (!ssid) throw new Error("No Wi-Fi ssid provided");
+  if (ssid === undefined) throw new Error("No Wi-Fi ssid provided");
```

Comment:
I don't think an empty SSID is allowed.
Also: Generally it's "SSID".

Reply:
I think , the jury is still out on whether an empty SSID should be allowed.

Reply:
Fixed comment, reverted check.